relational_operators/CreateIndexOperator.hpp

```diff
@@ -54,11 +59,14 @@ class CreateIndexOperator : public RelationalOperator {
    *
    * @param relation The relation to create index upon.
    * @param index_name The index to create.
+   * @param index_descriptions Set of index_descriptions associated with this index.
    **/
   CreateIndexOperator(CatalogRelation *relation,
-                      const std::string &index_name)
+                      const std::string index_name,
```

Comment:
Yeah sorry, I meant I fixed it my local branch, will push all the changes together once I am done. I usually go on marking all the suggested edits before I push them, so that I track all the ones that I have fixed.